lib/features/composer/presentation/widgets/email_editor_widget.dart

```diff
@@ -0,0 +1,75 @@
+
+import 'package:core/presentation/utils/html_transformer/html_utils.dart';
+import 'package:core/utils/app_logger.dart';
+import 'package:flutter/material.dart';
+import 'package:html_editor_enhanced/html_editor.dart';
+import 'package:tmail_ui_user/features/composer/presentation/composer_controller.dart';
+
+class EmailEditorWidget extends StatefulWidget {
+
+  final ComposerController controller;
+  final String content;
+  final TextDirection direction;
+
+  const EmailEditorWidget({
+    super.key,
+    required this.controller,
+    required this.content,
+    required this.direction,
+  });
+
+  @override
+  State<EmailEditorWidget> createState() => _EmailEditorState();
+}
+
+class _EmailEditorState extends State<EmailEditorWidget> {
+
+  late ComposerController _controller;
```

Comment:
you can write: `late ComposerController _controller = widget.controller`

Reply:
your writing is not encouraged by flutter. Although it's not wrong at runtime